microsoft-edge/extensions-chromium/publish/reply-user-reviews.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Respond to user reviews about your extension
+description: You can view your Microsoft Edge extension's reviews and ratings and respond to user reviews from the Partner Center dashboard.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 05/27/2021
+---
+# Respond to user reviews about your Microsoft Edge extension
```

Comment:
```suggestion
# Reply to user reviews about your Microsoft Edge extension
```
Consider "Reply".  If change it, change in multiple spots:  title:, in h1 here, and in toc.yml.  I added "reply" index entry synonym in toc.yml.  Find the word "reply" and "respond" in the page, and decide where to say which.  Recommend both.

Reply:
fyi i'm about to push a commit that updates the "title: " metadata

Reply:
fixing